app/views/courses/preview/_fees.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-fees">Fees</h2>
+
+  <% if course.fee_uk_eu.present? %>
+    <div class="body-text">
+      <table class="govuk-table">
+        <caption class="govuk-table__caption govuk-!-font-weight-regular govuk-!-margin-bottom-4">The course fees for 2019/2020 are as follows:</caption>
```

Comment:
Similar to the above comment about the apply URL, this could benefit from being a helper/decorator method that will, later, sort out the correct text to insert for the applicable recruitment cycle for this course.